packages/calcite-components/src/components/functional/InternalLabel.tsx

```diff
@@ -0,0 +1,61 @@
+import { TemplateResult } from "lit-html";
+import { h } from "@arcgis/lumina";
+
+interface InternalLabelProps {
+  alignmentCenter?: boolean;
+  labelText?: string;
+  onClick?: () => void;
+  required?: boolean;
+  noSpacingBottom?: boolean;
```

Comment:
These functional component props names could align better with components but it doesn't really matter since this is internal.

For example, `noSpacingBottom` could be `bottomSpaceDisabled` or `hideBottomSpace`? `alignmentCenter` could be `centerAlign`?